spyderlib/plugins/shortcuts.py

```diff
@@ -159,17 +63,12 @@ def data(self, index, role=Qt.DisplayRole):
         column = index.column()
         if role == Qt.DisplayRole:
             if column == CONTEXT:
-                return to_qvariant(shortcut.context)
+                return to_qvariant(_(shortcut.context.capitalize()))

```

Comment:
Only the modifiers are to be localized, right ? In that case the string should be parsed, each modifier translated, then the string would be rebuilt and displayed. I, don't see us providing a translation for every existing possible keybind.